modules/ROOT/pages/_partials/cookbook-dw/zip-arrays-together/inputs/payload.json

```diff
@@ -12,7 +12,7 @@
     }
   },
    {
-    "name":"cofee-table",
+    "name":"coffee-table",
```

Comment:
This an several others are code examples. In some cases, the change is ok. In others, it could mess up the example.